addon/styles/addon.scss

```diff
@@ -33,9 +33,11 @@
     flex-direction: row;
     flex: 1;
 
-    > .content {
+    > .main-container {
       flex: 1;
-      margin: 20px 0 0 20px;
+      .frost-object-details__content {
```

Comment:
Why have this at all?  Just let the content specify its own margin and drop this entirely